shopify/product-quiz-public-remix-ssr/api/models/shopifyProduct/actions/update.ts

```diff
@@ -11,4 +11,7 @@ export const onSuccess: ActionOnSuccess = async ({ params, record, logger, api,
   // Your logic goes here
 };
 
-export const options: ActionOptions = { actionType: "update" };
+export const options: ActionOptions = {
+  actionType: "update",
+  triggers: { api: true },
```

Comment:
Is this necessary? Took a quick scan through the codebase and couldn't find any use of `shopifyProduct.update()`